ACMS-4124: Updated README.md file. (#625)
acquia/drupal-recommended-project: README.md

## Other Drupal versions

> Drupal 11 is installed by default. To install Drupal 9, use the below command:

Reviewer: ```suggestion
Drupal 11 is installed by default. To install Drupal 10, use the below command:
```

```

> composer create-project acquia/drupal-recommended-project:^1

Reviewer: ```suggestion
composer create-project acquia/drupal-recommended-project:^2
```

```

> To install Drupal 10, use the below command:
> ```
> composer create-project acquia/drupal-recommended-project:^2
> ```

Reviewer: ```suggestion
To install Drupal 9, use the below command:
```
composer create-project acquia/drupal-recommended-project:^1
```
```
